pkg/cmd/taskrun/delete_test.go

```diff
@@ -432,6 +432,15 @@ func TestTaskRunDelete(t *testing.T) {
 			wantError:   false,
 			want:        "Are you sure you want to delete all TaskRuns related to ClusterTask \"random\" keeping 1 TaskRuns (y/n): All but 1 TaskRuns associated with ClusterTask \"random\" deleted in namespace \"ns\"\n",
 		},
+		{
+			name:        "Delete all with --ignore-running",
+			command:     []string{"delete", "--all", "-f", "-n", "ns", "--ignore-running"},
+			dynamic:     seeds[4].dynamicClient,
+			input:       seeds[4].pipelineClient,
+			inputStream: nil,
+			wantError:   false,
+			want:        "All TaskRuns deleted in namespace \"ns\"\n",
```

Comment:
should we handle the error message for this ? eg `All non-running TaskRuns have been deleted in namespace ns` :thinking:
Also can you please a test case where one taskrun is in running state and one in pending and after executing the command should delete the one which is not running?